tests/test_form_fields.py

```diff
@@ -26,4 +26,5 @@ def test_bound_form_with_instance():
 
 def test_choices():
     form = get_form()
-    assert form.base_fields["zero2"].choices == [(0, 'ZERO'), (1, 'ONE')]
+    assert form.base_fields["zero2"].choices == [(0, 'Zero'), (1, 'One')]
+    assert form.base_fields["int_enum"].choices == BLANK_CHOICE_DASH + [(0, 'foo'), (1, 'B')]
```

Comment:
No – the code for `zero2` actually _does_ change, albeit in a subtle way :)

`IntEnum` used to be imported directly from the stdlib.

However, now we're using our fancier IntEnum, which as the same `choices` behavior as `enumfields.Enum`, i.e. auto-titling the name for a choice label when no explicit label exists.

Before:

```python
from enum import Enum, IntEnum
```

After:

```python
from enumfields import Enum, IntEnum, EnumField, EnumIntegerField
```